### ISSUE TITLE
Review and publish Smoking in Public Places form and service page

### ISSUE THREAD

**Issue by desigonz** [cityofaustin/techstack#3703]
Once the form and service page are evaluated by stakeholders, I need to: 

- [x] get an internal copy-edit review from @Jo-Dwyer or @srigdon 

- [x] test email with stakeholders

- [x] Sam will establish the process on the backend, and determine the "owner of the content"

- [x] Send to translations

- [x] Publish

- [ ] ~~Incorporate translations~~ we have a separate issue for this

**Comment by ablangworthy27** [comment]
I am waiting for stakeholders to provide the prefered email address for submissions

**Comment by ablangworthy27** [comment]
I am now just waiting on translations.

@Jo-Dwyer Did you see that email I forwarded to you with info from Stephanie about the ownership issue? Also, would you be ok with me adding a hyperlink on the info page (and sending copy for translation, of course)? I'm thinking something like this:

_**Read the full ordinance** to learn more, or **report a violation**._
